README.md

```diff
@@ -1,2 +1,34 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+**NOTE**: The Migration procedure will come with a downtime; As we need to scale down the applications using the Volumes before migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1) Minimum ceph version should be Ceph-CSI supported
+2) Existing volumes should be dynamically provisioned.
+3) Stop the application pods that are consuming the flex volume(s) that need to be converted.
+4) `ceph-common`packages, for rbd ceph commands.
+
+## Steps to Mirgrate
+1) Create Ceph-CSI storageClass in which you want to migrate:
+   1) Example: `kubectl create -f https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml`
+2) 1) Create rbac required to run migrator pod
```

Comment:
Do these yamls exist? I don't see rbac.yaml

Reply:
oh the name is not correct right now, will change later